snooty/parser.py

```diff
@@ -234,9 +234,9 @@ def dispatch_visit(self, node: docutils.nodes.Node) -> None:
         elif isinstance(node, docutils.nodes.definition_list_item):
             self.state.append(n.DefinitionListItem((line,), [], []))
         elif isinstance(node, docutils.nodes.bullet_list):
-            self.state.append(n.ListNode((line,), [], False))
+            self.state.append(n.ListNode((line,), [], n.ListEnumType.unordered))
         elif isinstance(node, docutils.nodes.enumerated_list):
-            self.state.append(n.ListNode((line,), [], True))
+            self.state.append(n.ListNode((line,), [], node["enumtype"]))
```

Comment:
Is this valid? Isn't this passing a string where ListEnumType is expected? Do we need to use `n.ListEnumType[node["enumtype"]]`?